### ISSUE TITLE
[Feature requests] Proposed changes to improve the user experience and boost product adoption

### ISSUE THREAD

**Issue by ayush-vibrant** [vincelwt/chatgpt-mac#18]
I am a fan of the product and have some ideas for making the user experience even better, Here are a few of them

1. **Keyboard shortcut to take the user to the input bar**. (Similar to ⌘ + L in google chrome)
1.1. This would allow users to use the entire product with keyboard shortcuts, which aligns with the philosophy of creating the shortcut ⌘ + Shift + G.
3. **Single-click sign-in**
2.1. Sign-in currently requires 3-4 clicks, which is too many. We should reduce the number of clicks required for signing in to minimize friction. (I have engineering ideas on how to implement this)
4. **Add a button to the panel that opens the chatGPT web page**
3.1. Users frequently need to find results that would be more user-friendly if displayed on a web page (because they would have to scroll less, see the full content at once, etc.)

**Comment by vincelwt** [comment]
Hi,
Sure those sounds great :) Feel free to submit PRs if you know how to implement those. For signin, this shouldn't break the oauth logins.
Regarding 3. you can do right click on the icon, "Open in browser"

**Comment by ayush-vibrant** [comment]
![Screenshot 2022-12-06 at 12 24 02 PM](https://user-images.githubusercontent.com/28810075/205842075-923a5cfd-3538-497f-9522-7b49a27d0edc.png)

I am just getting these two options on right-click.

**Comment by vincelwt** [comment]
You're running an old version ;) try the latest release

**Comment by ayush-vibrant** [comment]
Hey, I am trying to run the project on my local machine so I can raise the PR.
But I am encountering the following error, can you help me here? 

```
Error occurred in handler for 'GUEST_VIEW_MANAGER_CALL': Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
    at node:electron/js2c/sandbox_bundle:57:414
    at EventEmitter.<anonymous> (node:electron/js2c/sandbox_bundle:45:375)
    at EventEmitter.emit (node:electron/js2c/sandbox_bundle:112:3752)
    at Object.onMessage (node:electron/js2c/sandbox_bundle:37:765)
```

This is in response to the command `npm start`

The complete response is 
```
> chatgpt-mac@0.0.5 start
> electron-forge start

✔ Checking your system
✔ Locating application
✔ Loading configuration
✔ Preparing native dependencies [0.4s]
✔ Running generateAssets hook

update-electron-app config looks good; aborting updates since app is in development mode
objc[49220]: Class WebSwapCGLLayer is implemented in both /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebCore.framework/Versions/A/Frameworks/libANGLE-shared.dylib (0x7ffb4161bec8) and /Users/ayush.sharma/IdeaProjects/chatgpt-mac/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libGLESv2.dylib (0x10f5e4328). One of the two will be used. Which one is undefined.
Menubar app is ready.
Error occurred in handler for 'GUEST_VIEW_MANAGER_CALL': Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
    at node:electron/js2c/sandbox_bundle:57:414
    at EventEmitter.<anonymous> (node:electron/js2c/sandbox_bundle:45:375)
    at EventEmitter.emit (node:electron/js2c/sandbox_bundle:112:3752)
    at Object.onMessage (node:electron/js2c/sandbox_bundle:37:765)
```

**Comment by ayush-vibrant** [comment]
Also please let me know in `forge.config.js` on line-20 to 22 - I need to put my own creds, right?
What is teamId being used for?

**Comment by ayush-vibrant** [comment]
Even though I am facing [this](https://github.com/vincelwt/chatgpt-mac/issues/18#issuecomment-1340362817) error, but the app starts in my menubar.

**Comment by ayush-vibrant** [comment]
Hey, I spent a reasonable amount of time, but could not make the 1st use-case work.
Things I have tried

1. Importing `webContents` and calling `webContents.focus()`
2. Made preload: true in `webPreferences` of `browserWindow` in `menubar` constructor.
3. Another alternative for the same 
```
globalShortcut.register("CommandOrControl+L", () => {
      if(window.isVisible()){
        console.log("command + L pressed when window is visible");
        mb.window.webContents.getFocusedWebContents().focus();
      } else {
        console.log("command + L pressed when window is not visible");
      }
    });
```

If you want, I can raise the PR and debug it together. But for now, I could not find the resolution 😢 

It's clear that I don't have much experience with electron. Would love to contribute to this with you. LMK how should we move forward on this?
